### PR TITLE
Add node command for static files dev server.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Add another option to run the local server. And allow to customize the sever port.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,11 +6,20 @@ help:
...
 
+port?=8000
+
 dev-static:
-	go run cmd/static/static.go
+	go run cmd/static/static.go $(port)
+
+
+dev-static-node:
+	npx http-server . -p $(port)
```
[the remark this answers is not in the document]
Comment (Hyperkid123) [reply Makefile:22]: It should not. We need the api prefix (/api/chrome-service/...) to be part of the file system. There are directories that contain a symlink to the `static` directory. Its a bit confusing but this CLI server cannot use proper routing.

**Review by BlakeHolifield — APPROVED** [review]

**Review by karelhala — COMMENTED** [review]
You could also include in the readme `npx http-server --port 8889` so folks doesn't have to run the go command/install go etc.

**Comment by Hyperkid123** [comment]
@karelhala there is a node alternative. Its described in the docs.

**Comment by karelhala** [comment]
@Hyperkid123 aaah, good point! Didn't notice it.

**Pull request merged**